### PR TITLE
Add service type

### PR DESCRIPTION
**Pull request by vramperez**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -307,6 +320,8 @@ The function `subscriptionCount` is an optional count function that only will be
 
 The function `getSpecification` should return a javascript object with the usage specification for the accounting unit according to the TMF635 usage management API ([TMF635 usage Management API](https://www.tmforum.org/resources/standard/tmf635-usage-management-api-rest-specification-r14-5-0/)).
 
+Finally, add the name of the developed accounting modulethe to the `config.modules` array in the `config.js` file (the accounting module name is the name of the file, e.g. `megabyte` and `megabyte.js`) and restart the Accounting Proxy.
```
Comment (fdelavega) [inline README.md]: there is a typo there 'modulethe''

File: cli-addService [hunk 2]
```diff
@@ -0,0 +1,63 @@
...
+if (program.args.length !== 3) {
+    logger.error('Invalid number of arguments. Usage: ./cli addService [-c] <publicPath> <URL> <appId>');
+    console.log(program.help());
+
+} else {
+
+    var contextBroker = program.contextBroker !== undefined ? true : false;
+    var publicPath = contextBroker ? program.args[2] : program.args[0];
+    var url = contextBroker ? program.args[0] : program.args[1];
+    var appId = contextBroker ? program.args[1] : program.args[2];
```
Comment (fdelavega) [inline cli-addService:33]: I don't understand why the external path and the URL are taken from different args if the service is a context broker

File: test/unit/db_sqlite_spec.js [hunk 3]
```diff
@@ -491,59 +457,56 @@ describe('Testing SQLITE database', function () {
...
+        var method = 'getAppId';
 
         it('should call the callback with error when db fails getting the appId', function (done) {
-            testGetAppId('Error', null, done);
+            var errorMsg = 'Error in database getting the appId.';
+
+            getTest(method, sentence, params, args, errorMsg, null, null, done);
         });
 
         it('should call the callback withou error when the is not an appId', function (done) {
```
Comment (fdelavega) [inline test/unit/db_sqlite_spec.js]: Typo "without"

**Comment by vramperez** [comment]
New commits

**Pull request merged**